arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -627,20 +591,20 @@ sealed class Option<out A> {
 
   inline fun <B> crosswalk(f: (A) -> Option<B>): Option<Option<B>> =
     when (this) {
-      is None -> empty<B>().map { empty() }
-      is Some -> f(t).map { Some(it) }
+      is None -> None.map { None }
```

Comment:
I guess this could be simplified to:
```suggestion
      is None -> this
```